src/app/components/task/Task.js

```diff
@@ -458,21 +476,25 @@ class Task extends Component {
       const { onClick } = props;
       return (
         <div className={styles['task-metadata-button']}>
-          <Button
+          <ButtonMain
             className="metadata-delete"
+            size="default"
+            variant="contained"
+            theme="error"
             onClick={() => {
               if (onClick) {
                 onClick();
               }
               this.submitDeleteTaskResponse(task.first_response.id);
             }}
-          >
-            <FormattedMessage
-              id="metadata.delete"
-              defaultMessage="Delete"
-              description="This is a label that appears on a button next to an item that the user can delete. The label indicates that if the user presses this button, the item will be deleted."
-            />
-          </Button>
+            label={
+              <FormattedMessage
+                id="metadata.delete"
+                defaultMessage="Delete"
+                description="This is a label that appears on a button next to an item that the user can delete. The label indicates that if the user presses this button, the item will be deleted."
+              />
+            }
+          />
```

Comment:
Let's replace these metadata messages with global edit, cancel, delete & save?

Reply:
ooh, good catch. One day I will see this before you do and do it automatically...but will update!